docs/organizations/using-gate-policies.md

```diff
@@ -0,0 +1,92 @@
+# Using gate policies
+
+Gate policies help you ensure that Codacy uses the same quality gates across your organization repositories.
```

Comment:
One thing that I don't see mentioned here (yet?): will there be a limit of how many gate policies it's possible to create for the same organization?

Reply:
I will double-check, but AFAIK, there's no limit.

Reply:
Confirmed! No limits of gate policies for an organization. 👍